packages/create-sitecore-jss/src/templates/angular/src/app/routing/layout/layout.component.ts

```diff
@@ -12,12 +12,16 @@ enum LayoutState {
   Error
 }
 
+interface RouteFields {
+  pageTitle: Field<string>;
```

Comment:
Should we also include `[name: string]: unknown;` here?
```suggestion
  [name: string]: unknown;
  pageTitle: Field<string>;
```

Reply:
It's possible, I added